code/modules/projectiles/guns/energy/magic.dm

```diff
@@ -224,6 +243,11 @@ obj/item/weapon/gun/energy/staff/focus/attack_self(mob/living/user as mob)
 	max_shots = 5
 	projectile_type = /obj/item/projectile/magic/fireball
 
+/obj/item/weapon/gun/energy/wand/fire/pickup(mob/living/user as mob)

```

Comment:
Instead of copy-pasting the same thing on all subtypes of `wand`, do this:

```dm
/obj/item/weapon/gun/energy/wand/pickup(mob/living/user)
	if(!user.is_wizard())
		user << "<span class='warning'>When you pick up \the [src], you can feel alarming pulses of heat rushing up your arms.</span>"
		user.make_dizzy(120)

/obj/item/weapon/gun/energy/wand/toy/pickup()
	return
```

Reply:
But then it's not personalized per wand type...